phys/module_surface_driver.F

```diff
@@ -3895,6 +3896,20 @@ SUBROUTINE surface_driver(                                         &
 
 
 !------------------------------------------------------------------
+        !For WRF-CTSM simulations, we would like the land model (CTSM)
+        ! to handle inland lake points.
+        ! Here, we are changing xland to include lake points, so that
+        ! CTSM can handle it.
+         DO j=j_start(ij),j_end(ij)
+            DO i=i_start(ij),i_end(ij)
+                xland_ctsm (i,j) = xland (i,j) 
+                !IF (lakemask(i,j).EQ.1. .AND. xland(i,j).GT.1.5 .AND. ht(i,j).GT.lake_min_elev) THEN
```

Comment:
Can we just remove this commented line?

Reply:
@dudhia Thanks! I just removed this extra commented-out line. I will go ahead and mark this as resolved.